hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/scm/net/TestNetworkTopologyImpl.java

```diff
@@ -64,10 +74,24 @@ public class TestNetworkTopologyImpl {
   private NetworkTopology cluster;
   private Node[] dataNodes;
   private Random random = new Random();
+  private static Consumer<List<? extends Node>> mockedShuffleOperation;
```

Comment:
```suggestion
  private Consumer<List<? extends Node>> mockedShuffleOperation;
```

I wonder why findbugs doesn't complain...